controllers/toolchainconfig/configuration.go

```diff
@@ -135,6 +135,14 @@ func (a AutoApprovalConfig) IsEnabled() bool {
 	return commonconfig.GetBool(a.approval.Enabled, false)
 }
 
+func (a AutoApprovalConfig) Domains() []string {
```

Comment:
Let's add a description to the function. And note that an empty slice means "all domains".

Reply:
got it

Reply:
done